content/WebApiHost/WebApiHost.csproj

```diff
@@ -14,9 +14,9 @@
     <PackageReference Include="Kralizek.Extensions.Logging" Version="2.0.1" />
     <PackageReference Include="Microsoft.AspNetCore.App" />
     <PackageReference Include="Microsoft.AspNetCore.Razor.Design" Version="2.2.0" PrivateAssets="All" />
-<!--#if (AddWcfDiscovery) -->
-    <PackageReference Include="EMG.Wcf.Discovery" Version="1.0.25" />
-<!--#endif-->
+    <!--#if (AddDiscoveryAdapter) -->
+    <PackageReference Include="EMG.Extensions.DependencyInjection.ServiceModel.DiscoveryAdapter" Version="1.3.0" />
+    <!--#endif-->
```

Comment:
If you are renaming AddWcfDiscovery symbols with AddDiscoveryAdapter, make sure you change it everywhere.

Also you need to update it in https://github.com/keghub/templates/tree/master/content/WebApiHost/.template.config

in [template.json](https://github.com/keghub/templates/blob/master/content/WebApiHost/.template.config/template.json)

and 

in [dotnetcli.host.json](https://github.com/keghub/templates/blob/master/content/WebApiHost/.template.config/dotnetcli.host.json)

files